dusty/systems/virtualbox/__init__.py

```diff
@@ -117,6 +117,11 @@ def initialize_docker_vm():
 def get_docker_vm_ip():
     return check_output_demoted(['docker-machine', 'ip', constants.VM_MACHINE_NAME]).rstrip()
 
+@memoized
+def get_docker_bridge_ip():
```

Comment:
since it's memoized, you could just call the function instead of passing it all over the place as an argument